src/_tutorials/server/get-started.md

```diff
@@ -102,28 +96,31 @@ To run the app from the command line, use the Dart VM by running the
 [`dart`](/tools/dart-vm) command:
 
 ```terminal
-> dart bin/main.dart
+$ dart bin/main.dart
 Hello world: 42!
 ```
 
 ## 7. Modify the app
 
 Let's customize the app you just created.
 
- 1. Edit `lib/cli.dart` to return a different result:
+ 1. Edit `lib/cli.dart` to calculate a different result. For example, divide the
+    previous value by two (see [Arithmetic operators][] for operator details):
 
-    ```dart
+    <?code-excerpt "misc/test/tutorial/get_started.dart (calculate)" replace="/~\/ 2/[!$&!]/g"?>
+    {% prettify dart %}
     int calculate() {
-      return -1;
+      return 6 * 7 [!~/ 2!];
```

Comment:
Following Filip's principle of making examples interesting rather than trivial, I changed the return expression (which in this case exposes the reader to the `~/` operator, which they might not know exists). Let me know if this works for you!